client/app/components/dynamic-table/index.js

```diff
@@ -34,11 +34,11 @@ function sortRows(rows, orderBy) {
     for (let i = 0; i < orderBy.length; i += 1) {
       va = a[orderBy[i].name];
       vb = b[orderBy[i].name];
-      if (va == undefined || (va < vb)) {
+      if ((va === undefined) || (va < vb)) {
         // if a < b - we should return -1, but take in account direction
         return orderBy[i].direction * -1;
       }
-      if ((va > vb) || vb == undefined) {
+      if ((va > vb) || (vb === undefined)) {
```

Comment:
Actually if you do `===` it will not work as intended :) You need to add comparison with `null` now as well.

Reply:
Will do :+1: